pkg/rid/store/cockroach/store.go

```diff
@@ -47,12 +47,13 @@ type Store struct {
 // Interact implements store.Interactor interface.
 func (s *Store) Interact(ctx context.Context) (repos.Repository, error) {
 	logger := logging.WithValuesFromContext(ctx, s.logger)
+	storeVersion, err := s.GetVersion(ctx)
+	if err != nil {
+		return nil, err
```

Comment:
Because Go doesn't natively support stacktrace production, please generally add context when propagating errors by, e.g.: `return nil, stacktrace.Propagate(err, "Error determining database RID schema version")`.  This command is supported by adding `"github.com/palantir/stacktrace"` to the imports.  (This is a new general request and hasn't yet been documented in developer guidelines)

Reply:
Done